### PR TITLE
Stats (Editor Window)

### PR DESCRIPTION
**Pull request by ah1053**
1. Server Stats (Editor Window)
2. Updated EdgeMultiplay Package.

### CODE REVIEW

File: EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs [hunk 1]
```diff
@@ -78,11 +81,67 @@ public static void ReportBug()
...
+            else
+            {
+                string fqdn = await GetMobiledgeXAppFQDN();
+                if (fqdn == "")
+                {
+                    Debug.LogError("Make sure MobiledgeX Settings AppDefs and Region are correct, Check the console for more details.");
+                }
+                else
+                {
+                    Application.OpenURL("http://" + fqdn + ":7776");
```
Comment (franklin-huang-mobiledgex) [inline EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs:100]: anyway we can make this https in your node server?

File: EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs [hunk 2]
```diff
@@ -78,11 +81,67 @@ public static void ReportBug()
...
+            try
+            {
+                await integration.RegisterAndFindCloudlet();
+                if (integration.FindCloudletReply.status == FindCloudletReply.FindStatus.FIND_FOUND)
+                {
+                    return integration.FindCloudletReply.fqdn;
+                }
+                else
+                {
+                    return "";
```
Comment (franklin-huang-mobiledgex) [inline EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs:127]: hmm. If you return an empty string, the OpenURL will just get a 404 error right? Should probably log this at least

**Review by franklin-huang-mobiledgex — APPROVED** [review]

**Review by brucearmstrong — APPROVED** [review]

File: EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs (hunk 2 above)
Comment (ah1053) [reply EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs:127]: In the code, I check and Log the error already

Comment (franklin-huang-mobiledgex) [reply EdgeMultiplay/Editor/EdgeMultiplayEditorWindow.cs:127]: Gotcha. I see that you tell them to check console and you have logs for RegisterClientException and FindCloudletException. Could you log the FindCloudletStatus here as well then?

**Pull request merged**